src/EditorFeatures/Test2/FindReferences/FindReferencesTests.Tuples.vb

```diff
@@ -224,11 +229,122 @@ namespace System
     }
 
         ]]>
+
+            <%= tuple2 %>
+        </Document>
+    </Project>
+</Workspace>
+            Await TestAPIAndFeature(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestTupleFieldItem01() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document><![CDATA[
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var x = ({|Definition:Alice|}:1, Bob: 2);
+            var y = (Alice:1, Bob: 2);
+
+            var z = x.$$[|Item1|];
+        }
+    }
+
+        ]]>
+
+            <%= tuple2 %>
         </Document>
     </Project>
 </Workspace>
             Await TestAPIAndFeature(input)
         End Function
 
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestTupleFieldItem02() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document><![CDATA[
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var x = ($$[|{|Definition:Alice|}|]:1, Bob: 2);
+            var y = ([|Alice|]:1, Bob: 2);
+
+            var z = x.Item1;
+        }
+    }
+
+        ]]>
+
+            <%= tuple2 %>
+        </Document>
+    </Project>
+</Workspace>
+            Await TestAPIAndFeature(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestTupleFieldItem03() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document><![CDATA[
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var x = ({|Definition:1|}, Bob: 2);
+            var y = (Alice:1, Bob: 2);
+
+            var z = x.$$[|Item1|];
+            var z1 = x.[|Item1|];
+        }
+    }
+
+        ]]>
+
+            <%= tuple2 %>
+        </Document>
+    </Project>
+</Workspace>
+            Await TestAPIAndFeature(input)
+        End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestTupleFieldItem04() As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document><![CDATA[
```

Comment:
CData is convenient since you can put more stuff in them without having to escape `&lt;` and stuff like that. After that I just copied the test and changed the content where needed. 
I think removing CDATA where not needed is not very useful.